Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -371,6 +378,18 @@ void ApplyIntegretionLibraries (string socialPlatform, bool remove)
 
 				EditorGUI.EndDisabledGroup();
 				break;
+			case "gameCenter":
+#if UNITY_IOS
```

Comment:
@EugeneButusov, we have similar precendents in SoomlaStore, where iOS and Android params are visible for all platforms. I think it's less confusing, than magically disappearing settings, when a developer changes target platform before building. Unlike cocos2d-x, Unity3d projects are the same for all platforms, and we need adapt to this.